modules/functional-test/src/main/java/com/intuit/wasabi/tests/perftest/PerfTest.java

```diff
@@ -0,0 +1,127 @@
+import org.testng.annotations.Test;
+
+import com.intuit.wasabi.tests.library.TestBase;
+import com.intuit.wasabi.tests.library.util.Constants;
+import com.intuit.wasabi.tests.model.Application;
+import com.intuit.wasabi.tests.model.Bucket;
+import com.intuit.wasabi.tests.model.Experiment;
+import com.intuit.wasabi.tests.model.factory.ApplicationFactory;
+import com.intuit.wasabi.tests.model.factory.BucketFactory;
+import com.intuit.wasabi.tests.model.factory.ExperimentFactory;
+
+public class PerfTest extends TestBase {
+
+    // lists that holds the experiments of test applications
+    List<Experiment> experimentListOfCTG = new ArrayList<Experiment>();
+    List<Experiment> experimentListOfQBO = new ArrayList<Experiment>();
+
+    // create performance test Applications
+    Application ctgApplication = ApplicationFactory.createApplication().setName("wasabiloadtest_CTG");
+    Application qboApplication = ApplicationFactory.createApplication().setName("wasabiloadtest_QBO_Events");
+
+    @BeforeClass
+    public void cleanUp() {
+        List<Experiment> experimentsListOfCTG = getApplicationExperiments(ctgApplication);
+
+        for (Experiment exp : experimentsListOfCTG) {
+
+            // pause the experiment
+            exp.state = Constants.EXPERIMENT_STATE_PAUSED;
+            putExperiment(exp);
+
+            // terminate the experiment
+            exp.state = Constants.EXPERIMENT_STATE_TERMINATED;
+            putExperiment(exp);
+
+            // delete the experiment
+            deleteExperiment(exp);
+        }
+
+        List<Experiment> experimentsListOfQBO = getApplicationExperiments(qboApplication);
+
+        for (Experiment exp : experimentsListOfQBO) {
+
+            // pause the experiment
+            exp.state = Constants.EXPERIMENT_STATE_PAUSED;
+            putExperiment(exp);
+
+            // terminate the experiment
+            exp.state = Constants.EXPERIMENT_STATE_TERMINATED;
+            putExperiment(exp);
+
+            // delete the experiment
+            deleteExperiment(exp);
+        }
+
+    }
+
+    @Test
+    public void prepareData() {
+
+        // now lets create experiments for each applications
+        for (int i = 1; i <= 15; i++) {
+            Experiment exp = ExperimentFactory.createExperiment("wasabi-perftest-ctgapp-exp" + i)
+                    .setApplication(ctgApplication);
+            experimentListOfCTG.add(exp);
+        }
+
+        experimentListOfCTG = postExperiments(experimentListOfCTG);
+
+        // now lets create experiments for each applications
+        for (int i = 1; i <= 15; i++) {
+            Experiment exp = ExperimentFactory.createExperiment("wasabi-perftest-qboapp-exp" + i)
+                    .setApplication(qboApplication);
+            experimentListOfQBO.add(exp);
+        }
+
+        // lets create experiment aaa that is used for single assignment
+        Experiment exp = ExperimentFactory.createExperiment("aaa").setApplication(qboApplication);
```

Comment:
Can we use a more meaningful experiment name? It will help us monitor/observe/remember better while testing. Also, suggest moving the name to a string constant and using it here.